clang/test/CodeGen/2005-01-02-ConstantInits.c

```diff
@@ -1,10 +1,24 @@
-// RUN: %clang_cc1 %s -emit-llvm -o -
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --function-signature --check-globals --global-value-regex "@.+"
+// RUN: %clang_cc1 -triple=x86_64-unknown-linux %s -emit-llvm -o - | FileCheck %s
 
 // This tests all kinds of hard cases with initializers and
 // array subscripts.  This corresponds to PR487.
 
 struct X { int a[2]; };
 
+//.
```

Comment:
Is this intentional?

Reply:
upstream update_test_checks added it.

Reply:
Interesting, it's a magic marker for --check-globals. Has to be a comment because otherwise you change the input and that creates idempotency issues when you come to regenerate, and I guess it helps disambiguate from `//` which is already used to separate different sets of CHECK lines for the same function.